joomla/joomla-cms
# [Fix #5635] Adding a significant message when passwords do not match

See https://github.com/joomla/joomla-cms/issues/5635
Test instructions:
Back-end
User Manager=>Add New User
Fill Name, login name, Password, email correctly but do not enter the same password in 'confirm password' field and click on save button.

After patch we will now get (same message as in other similar conditions in frontend and for profile edit)
![screen shot 2015-05-10 at 08 40 46](https://cloud.githubusercontent.com/assets/869724/7553321/dc5538da-f6f1-11e4-96ed-f4a996f7925c.png)
